x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_definitions/entity_descriptions/universal.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { EntityDescription } from '../types';
+import { collectValues as collect } from './field_utils';
+
+export const UNIVERSAL_DEFINITION_VERSION = '1.0.0';
+export const UNIVERSAL_IDENTITY_FIELD = 'related.entity';
+
+const entityMetadataExtractorProcessor = {
```

Comment:
Would this be better placed in its own file in `elasticsearch_assets/ingest_processor_steps`?

Reply:
My reasoning for leaving it here is because this is only needed for the universal entity, so it's the universal entity maintainers' responsibility.